docs/next/guides/building-and-testing/modules.md

```diff
@@ -143,129 +145,483 @@ Elements to be imported manually on demand:
+registerPlugin(AutoColumnSize);
+registerPlugin(Autofill);
+registerPlugin(AutoRowSize);
+registerPlugin(BindRowsWithHeaders);
+registerPlugin(CollapsibleColumns);
+registerPlugin(ColumnSorting);
+registerPlugin(ColumnSummary);
+registerPlugin(Comments);
+registerPlugin(ContextMenu);
+registerPlugin(CopyPaste);
+registerPlugin(CustomBorders);
+registerPlugin(DragToScroll);
+registerPlugin(DropdownMenu);
+registerPlugin(ExportFile);
+registerPlugin(Filters);
+registerPlugin(Formulas);
+registerPlugin(HeaderTooltips);
+registerPlugin(HiddenColumns);
+registerPlugin(HiddenRows);
+registerPlugin(ManualColumnFreeze);
+registerPlugin(ManualColumnMove);
+registerPlugin(ManualColumnResize);
+registerPlugin(ManualRowMove);
+registerPlugin(ManualRowResize);
+registerPlugin(MergeCells);
+registerPlugin(MultiColumnSorting);
+registerPlugin(MultipleSelectionHandles);
+registerPlugin(NestedHeaders);
+registerPlugin(NestedRows);
+registerPlugin(ObserveChanges);
+registerPlugin(PersistentState);
+registerPlugin(Search);
+registerPlugin(TouchScroll);
+registerPlugin(TrimRows);
+registerPlugin(UndoRedo);
+// or, register all plugins at once
+registerAllPlugins();
+
+// or, register all Handsontable modules at once
+registerAllModules();
+
+// create a Handsontable instance
+new Handsontable(container, {});
+```
+:::
+
+## Importing editors
+
+To use a specific editor you need to import it alongside the registering method. For instance, let's try adding the password editor.
+
+Start with importing the base, `PasswordEditor` and the `registerEditor` method.
+
+```js
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+// import the method for registering an editor
+// import the PasswordEditor
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+```
+
+Afterwards you need to use the `registerEditor` method to register `PasswordEditor`:
+
+```js
+// register the editor before using it
+registerEditor(PasswordEditor);
+```
+
+Now, you can use `PasswordEditor`, the full example looks like this:
+
+```js
+import Handsontable from 'handsontable/base';
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+
+registerEditor(PasswordEditor);
+
+// use the editor as needed
+new Handsontable(container, {
+  columns: [
+    {
+      editor: 'password',
+    },
+  ]
+  // rest of the settings
+});
+```
+
+And that is all! You can use the password editor!
+
+#### Importing all editors at once
+
+You can quickly import and register all editors at once:
+
+```js
+// import all editors
+import {
+  registerEditor,
+  AutocompleteEditor,
+  BaseEditor,
+  CheckboxEditor,
+  DateEditor,
+  DropdownEditor,
+  HandsontableEditor,
+  NumericEditor,
+  PasswordEditor,
+  SelectEditor,
+  TextEditor,
+} from 'handsontable/editors';
+
+// import the `registerAllEditors()` function
+import { registerAllEditors } from 'handsontable/registry';
+
+// register all editors
+registerAllEditors();
```

Comment:
```suggestion
// import the `registerAllEditors()` function
import { registerAllEditors } from 'handsontable/registry';

// register all editors
registerAllEditors();
```
There is no need to import the editors when you're using the `registerAllEditors` function.